command/alias.go

```diff
@@ -16,21 +16,34 @@ func init() {
 	aliasCmd.AddCommand(aliasSetCmd)
 	aliasCmd.AddCommand(aliasListCmd)
 	aliasCmd.AddCommand(aliasDeleteCmd)
+
+	aliasSetCmd.Flags().BoolP("shell", "s", false, "Whether the alias should be passed to a shell")
 }
 
 var aliasCmd = &cobra.Command{
 	Use:   "alias",
-	Short: "Create shortcuts for gh commands",
+	Short: "Create command shortcuts",
+	Long: heredoc.Doc(`
+	Aliases can be used to make shortcuts for gh commands or to compose multiple commands.
+
+	Run "gh help alias set" to learn more.
+	`),
 }
 
 var aliasSetCmd = &cobra.Command{
 	Use:   "set <alias> <expansion>",
 	Short: "Create a shortcut for a gh command",
-	Long: `Declare a word as a command alias that will expand to the specified command.
+	Long: heredoc.Doc(`
+	Declare a word as a command alias that will expand to the specified command(s).
 
-The expansion may specify additional arguments and flags. If the expansion
-includes positional placeholders such as '$1', '$2', etc., any extra arguments
-that follow the invocation of an alias will be inserted appropriately.`,
+	The expansion may specify additional arguments and flags. If the expansion
+	includes positional placeholders such as '$1', '$2', etc., any extra arguments
+	that follow the invocation of an alias will be inserted appropriately.
+
+	If --shell is specified, the alias will be run through a shell. This allows you to compose
+	commands with | or redirect output with >.
```

Comment:
Could we quote bits that are not prose?
```suggestion
	If '--shell' is specified, the alias will be run through a shell. This allows you to compose
	commands with "|" or redirect output with ">".
```